etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -40,20 +40,26 @@ stringData:
     - repoReader
 {{- end }}
 
+# when registering a cluster with a separate enterprise server
+{{ if not .Values.pachd.enterpriseLicenseKey }}
```

Comment:
Is there ever a case where you're not specifying the enterprise license key and you wouldn't nee enterpriseServerAddress and enterpriseRootToken?

Reply:
Good question. The only enterprise activation case where you don't require an enterprise key is when you already have an enterprise license key activated on another cluster, and you want to register with that cluster to inherit it's enterprise capabilities. 

That's the reason `activate == true && licenseKey == ""` is implying registering with another cluster here